content/cybersecurity/concepts/blue-team/blue-team.md

```diff
@@ -0,0 +1,31 @@
+---
+Title: 'Blue Team'
+Description: 'Blue team typically consists of incident response professionals who try to stop sophisticated attacks.'
+Subjects:
+  - 'Cybersecurity'
+Tags:
+  - 'Ethical Hacking'
```

Comment:
We still want the "Cybersecurity" tag in addition to this. We like to have at least two tags.